app/views/widgets/lists.html

```diff
@@ -19,6 +19,20 @@
 
 {% endmacro %}
 
+{% macro taskStatus(content, title, status, previousTaskStatus) %}
```

Comment:
This macro isn't really defining a list so we'd be better to not put it in the`lists.html` file. Perhaps we can create a new file specifically for this macro.